distinct-elements.js

```diff
@@ -0,0 +1,11 @@
+var array = [10, 3, 5, 12, 5, 3, 9, 1];
+
+for (i = 0; i < array.length; i++) {
+    count = 0;
+    for (j = 0; j < array.length; j++) {
+        if (array[i] == array[j]) {
+            count++;
+        }
+    }
+    console.log(array[i] + "-" + count);
+}
```

Comment:
Does not work as expected.  
10-1
3-2
5-2
12-1
5-2
3-2
9-1
1-1

You can see that 5-2 is coming twice.

Reply:
Can you solve this out using an additional variable ie element_count ?

Reply:
yes , i can get the desired output by printing it soon after the count rather than saving in another variable.